pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,6 +126,32 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+func (pl *VolumeBinding) isSchedulableAfterStorageClassChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
```

Comment:
mention that volume binding mode is immutable and we don't have to take the update there into consideration.

Reply:
You overlooked this ^

```suggestion
// isSchedulableAfterStorageClassChange checks whether an StorageClass event might make a Pod schedulable or not. 
// Any StorageClass addition and a StorageClass update to allowedTopologies 
// might make a Pod schedulable. 
// Note that an update to volume binding mode is not allowed and we don't have to consider while examining the update event.
func (pl *VolumeBinding) isSchedulableAfterStorageClassChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
```